install_config/persistent_storage/persistent_storage_glusterfs.adoc

```diff
@@ -193,7 +181,14 @@ Gluster server administrators using normal POSIX compliant security.
 By default, SELinux does not allow writing from a pod to a remote Gluster
 server.
 
-To enable writing to GlusterFS volumes with SELinux enforcing on each node, run:
+To enable writing to GlusterFS volumes with SELinux enforcing first install
+the`docker-selinux on each node:
+
+----
+# yum install docker-selinux
```

Comment:
@swagiaal @erinboyd AFAICT, this package is installed when installing the docker RPM already, at least for RHEL 7.2 and Fedora 22. Do we need to call it out separately here?